lwt/core_lwt.ml

```diff
@@ -0,0 +1,30 @@
+open Core_kernel.Std
```

Comment:
Actually, the things are completely opposite here. Usage of the cooperative frameworks like `lwt` or `async` allows one to solve problems with availability in a high load environment. When cooperative thread performs a blocking call (I'm talking about a call lifted to `lwt` not to the real system call) it  will automatically yield to another thread. 

1) No, each connection to bap is very cheap, and bap can hold tons of its without worry. We should only worry on memory footprint and actual disassembly. For that case, I've added push back mechanism, that will not accept any new requests, until the older one are resolved.

2) Nope. It will respond and very fast.